### PR TITLE
Added QBCore.Shared.Vehicles.categoryLabel support

### PR DESCRIPTION
**Pull request by r0adra93**
Added an if statement to check for categorylabel and to use categoryLabel when present.

**Describe Pull request**
First, make sure you've read and are following the contribution guidelines and style guide and your code reflects that.
Write up a clear and concise description of what your pull request adds or fixes and if it's an added feature explain why you think it should be included in the core.

If your PR is to fix an issue mention that issue here

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes
- Does your code fit the style guidelines? [yes

### CODE REVIEW

**Comment by Donnyjohn1** [comment]
can you re add to the new vehicle shops lua please, my server got deleted by hosting and now i dont have it :(

**Comment by Donnyjohn1** [comment]
also breaks the admin menu, no longer shows players, or vehicles list
image
image

File: client/client.lua [hunk 1]
```diff
@@ -1,4 +1,4 @@
-local banlength = nil
+local banlength
```
Comment (charj) [inline client/client.lua:1]: This change appears to be unrelated, was this intentional?

Comment (charj) [reply client/client.lua:1]: Linter change. 
`local banlength` would be nil by default anyway

**Comment by r0adra93** [comment]
> can you re add to the new vehicle shops lua please, my server got deleted by hosting and now i dont have it :(

I readded support for categoryLabel it's in PR 7 waiting to be tested and merged

**Comment by Donnyjohn1** [comment]
> > can you re add to the new vehicle shops lua please, my server got deleted by hosting and now i dont have it :(
> 
> I readded support for categoryLabel it's in PR 7 waiting to be tested and merged

you sir are my favorite thankyou
